nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -114,15 +115,25 @@ class TensorAggregatorBase:
     aggregate them in terms of NNCFCollectorTensorProcessor operations.
     """
 
-    def __init__(self, tensor_processor: NNCFCollectorTensorProcessor, num_samples: Optional[int] = None):
+    def __init__(
+        self,
+        tensor_processor: NNCFCollectorTensorProcessor,
+        aggregation_axes: Optional[Tuple[int, ...]] = None,
+        num_samples: Optional[int] = None,
+    ):
         """
         :param tensor_processor: Backend-specific tensor processor.
+        :param aggregation_axes: Axes along which to operate.
+            Registered statistics are stacked along zero axis,
+            axes >=1 correspond to recieved statistic axes shifted left by 1.
```

Comment:
What is the difference between `reduction_axes` and `aggregation_axes` beside the names?

Reply:
Although these do have the same underlying type right now, they do have different meaning and impact different stages in stat collection, so different names (and even different type aliases) are justified in order not to mix them up. Maybe some more clarification or examples in the docstring are in order to highlight the difference in meaning.

Reply:
`reduction_axes` are used during samples reduction step inside a reducer, 
`aggregation_axes` are used during aggregation step inside an aggregator